### PR TITLE
Update README to state that for pubsub topic must exist before publish

### PR DESCRIPTION
**Pull request by davidpadbury**
Resolves issue #1052

### CODE REVIEW

**Comment by davidpadbury** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -230,11 +230,10 @@ var pubsub = gcloud.pubsub({
   keyFilename: '/path/to/keyfile.json'
 });
 
-// Reference a topic.
+// Reference a topic that has been previously created
```
Comment (stephenplusplus) [inline README.md]: Good call adding this. Total nitpick, can you pop a `.` at the end of the line?

**Comment by davidpadbury** [comment]
@stephenplusplus done in 91d4382.

**Comment by stephenplusplus** [comment]
Looks good, thanks again.

**Pull request merged**